### ISSUE TITLE
Spider gbXML Viewer 'Maevia' R15.25: Add Construction

### ISSUE THREAD

**Issue by mdengusiak** [ladybug-tools/spider-gbxml-tools#35]
@theo-armour as discussed here is 
1. Add Construction - visualisation for each BuildingElement

```
  <Construction id="BasicRoof_SIM_EXT_SLD_RoofDA01">
    <U-value unit="WPerSquareMeterK">0.18</U-value>
    <Absorptance unit="Fraction" type="ExtIR">0</Absorptance>
    <Roughness value="Rough" />
    <LayerId layerIdRef="layer-Plasterboard-Air50mmceiling-ConcreteDeck-membrane-Insulation1" />
    <Name>Basic Roof : SIM_EXT_SLD_Roof DA01</Name>
  </Construction>
```

as starting point Construction : 
- Name
- U-value

then we follow up with 

```
  <WindowType id="window-WinInst_SIM_EXT_GLZ">
    <Name>WinInst : SIM_EXT_GLZ</Name>
    <Description>Standard Dbl Glazed</Description>
    <U-value unit="WPerSquareMeterK">1.527</U-value>
    <SolarHeatGainCoeff unit="Fraction">0.401</SolarHeatGainCoeff>
```
WindowType:
- Name
- U-value 
- SolarHeatGainCoeff 

test file here:
[BHoMgbXML_Constructions.zip](https://github.com/ladybug-tools/spider-gbxml-tools/files/2923244/BHoMgbXML_Constructions.zip)

**Comment by mdengusiak** [comment]
Also here is for reference working file that run in OpenStudio
[BHoMgbXML_fixed_runningOpenStudio.zip](https://github.com/ladybug-tools/spider-gbxml-tools/files/2923333/BHoMgbXML_fixed_runningOpenStudio.zip)

**Comment by mdengusiak** [comment]
@theo-armour  
wooow amazing to see Construction entering Spider :)

**Comment by mdengusiak** [comment]
@theo-armour  to be included in Surface Attribute Module, so in Spider Fixer as well

**Comment by theo-armour** [comment]
@mdengusiak 
New view menu items added to fixer

**Comment by theo-armour** [comment]
OK to close??

**Comment by michaldengusiak** [comment]
yes we have this now included in  Spider gbXML Viewer 'Maevia' 0.17.05-0

![image](https://user-images.githubusercontent.com/26113670/67365244-2a350600-f569-11e9-9c18-a80b8a961114.png)
